runtime/src/lib.rs

```diff
@@ -1023,21 +1023,12 @@ parameter_types! {
     pub const MinNumberOfExtraCandidates: u32 = 0;
     pub const AnnouncingPeriodDuration: BlockNumber = 300;
     pub const IdlePeriodDuration: BlockNumber = 1;
+    pub const CouncilSize: u32 = 3;
     pub const MinCandidateStake: Balance = dollars!(10_000);
     pub const ElectedMemberRewardPeriod: BlockNumber = 33;
     pub const BudgetRefillPeriod: BlockNumber = 33;
 }
 
-#[cfg(feature = "runtime-benchmarks")]
-parameter_types! {
-    pub const CouncilSize: u32 = 3;
-}
-
-#[cfg(feature = "playground-runtime")]
-parameter_types! {
-    pub const CouncilSize: u32 = 1;
-}
-
```

Comment:
Hmm, it looks like for some reason this merge is reverting all changes introduced in https://github.com/Joystream/joystream/pull/4428

Reply:
Possibly related to https://github.com/Joystream/joystream/commit/dabaa27f3ed5efb406d7560e7205ebe1dc0aee3f

Reply:
Hmm thats odd. Thanks for spotting this. I'll merge locally and open a separate PR in this case. Perhaps I will see merge conflicts and make the right choice.

Reply:
fixed in https://github.com/Joystream/joystream/pull/4465